src/components/Scenarios/ScenariosColumns.jsx

```diff
@@ -0,0 +1,80 @@
+import strings from 'lang';
+import { transformations, formatSeconds } from 'utility';
+import { inflictorWithValue } from 'components/Visualizations';
+
+const computeWinRate = row => (row.wins / row.games);
+
+const getTimeRange = (field, metadata) => {
+  let lower;
+  if (metadata.indexOf(field) !== 0) {
```

Comment:
do you want -1 here?

Reply:
no the method can never return -1. the logic is: "if the time value is not the first one in the array then the lower time range must be 0:00. else the lower range must be the one before this.